orchagent/main.cpp

```diff
@@ -33,6 +33,8 @@ extern "C" {
 using namespace std;
 using namespace swss;
 
+
+
```

Comment:
@kcudnik do we need to remove this debug stuff?

Reply:
removed